packages/lightning-core/helpers/paymentRequest.js

```diff
@@ -5,5 +5,5 @@ export const decoratePaymentRequest = (pr) => {
 }
 
 export const sanitizePaymentRequest = (pr) => {
-  return pr.replace(prefix, '')
+  return pr.replace(/lightning:(\/\/)?/g, '')
```

Comment:
Can you add comment explaining the possible prefix matches?

Reply:
I just found the two above (`lightning:` and `lightning://`) so they are the only possible matches

Reply:
I meant to add it as a comment in the code above that line, something like:
 `// Payment requests should start with either lightning: or lightning://`

Reply:
oh sorry for my misunderstanding. I'll do it rn

Reply:
Is there any documentation for the `lightning://` prefix? I could only find https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md which says not to use `lightning://`

Reply:
Our usage of the `lightning://` prefix actually predates BOLT 11 itself.